### PR TITLE
Add functions using PyCall

### PR DESCRIPTION
**Pull request by ShivamNegi**

### CODE REVIEW

File: src/retriever.jl [hunk 1]
```diff
@@ -1,5 +1,50 @@
...
+
+function install_json(dataset; table_name::Bool=nothing, compile::Bool=false, debug::Bool=false, quite::Bool=false, use_cache::Bool=true)
+    rt.install_json(dataset, table_name, compile, debug, quite, use_cache)
+end
+
+function install_xml(dataset; table_name::Bool=nothing, compile::Bool=false, debug::Bool=false, quite::Bool=false, use_cache::Bool=true)
+    rt.install_xml(dataset, table_name, compile, debug, quite, use_cache)
+end
+
+function reset_retriever(;scope::AbstractString="all")
```
Comment (henrykironde) [inline src/retriever.jl]: (;scope
I think we need space and is the `;` required?

File: src/retriever.jl [hunk 2]
```diff
@@ -1,5 +1,50 @@
 module retriever
+using PyCall
 
+@pyimport retriever as rt
 # package code goes here
```
Comment (henrykironde) [inline src/retriever.jl]: # package code goes here
not required

**Review by henrykironde — COMMENTED** [review]
I think the functions look good.
We should refactor and make the code length shorter.

File: src/retriever.jl (hunk 1 above)
Comment (ShivamNegi) [reply src/retriever.jl]: Yes, `;` is used to separate [keyword arguments](https://docs.julialang.org/en/release-0.5/manual/functions/).

**Comment by henrykironde** [comment]
I was meaning the lines of code are long, I will look for some rules on that.

**Comment by henrykironde** [comment]
I think If you use ATOM or its clone http://junolab.org/ you can do some auto refactoring and them add some manual changes

**Comment by ShivamNegi** [comment]
I have indented similar to PEP standards, keeping the line length <= 80. Couldn't find anything like 'standards' for Julia.

**Comment by henrykironde** [comment]
Okey thanks, let me know when you update

**Comment by henrykironde** [comment]
@ShivamNegi, I think I have this same commit, I will test and see once again

**Comment by henrykironde** [comment]
```
julia> include("src/retriever.jl")
retriever

julia> retriever.install_sqlite("iris")
=> Installing iris

Downloading bezdekIris.data...
100%  0 seconds Creating table iris_table...
Progress: 151 / 151 rows inserted into iris_table totaling 151:


julia> retriever.install_sqlite("iris", use_cache=false, debug=true, quite=false)
=> Installing iris
Creating table iris_table...
['sepal_length', 'sepal_width', 'petal_length', 'petal_width', 'class']
CREATE TABLE iris_table (sepal_length REAL, sepal_width REAL, petal_length REAL, petal_width REAL, class TEXT(20) );

Saving a copy of bezdekIris.data...

Downloading bezdekIris.data...
100%  0 seconds INSERT INTO iris_table (sepal_length, sepal_width, petal_length, petal_width, class) VALUES (?, ?, ?, ?, ?)
Progress: 151 / 151 rows inserted into iris_table totaling 151:


julia> 
```

File: REQUIRE [hunk 3]
```diff
@@ -1 +1,3 @@
 julia 0.5
+PyCall 1.12
+retriever 2.0
```
Comment (henrykironde) [inline REQUIRE]: remove retriever in this case or make it retriever 2.1

Comment (ShivamNegi) [reply REQUIRE]: Removed `retriever` from REQUIRE.

**Pull request merged**